normal_speed: 2.5
turbo_speed: 4.0
accel_limit: 10.0
maxTurnRate: 0.25
commandInterval: 0.05
speed_to_erpm_gain: -5133
speed_to_erpm_offset: 180.0
erpm_speed_limit: 22000
steering_to_servo_gain: -1.8
steering_to_servo_offset: 0.525
servo_min: 0.0
servo_max: 1.0
wheelbase: 0.324
